inc/certificate.class.php

```diff
@@ -48,8 +48,8 @@ class Certificate extends CommonDBTM {
    protected $usenotepad       = true;
 
    /**
-    * @param int $nb
-    * @return string|translated
+    * {@inheritDoc}
```

Comment:
NOK since $nb will not be documented on output

Reply:
OK I removed the complete block as it will be inherited by default.